docker-compose.yml

```diff
@@ -1,19 +1,25 @@
-version: '2'
+version: '3'
 
 services:
   redis:
     image: redis
   db:
-    image: postgres:10
+    image: postgres
```

Comment:
Why unpin Postgres here?

Reply:
to make it closer to https://docs.docker.com/compose/django/